userguide/services.rst

```diff
@@ -280,10 +280,7 @@ summarizes the available options.
    | DNS Forwarder          | string         | IP address of DNS forwarder; required for recursive queries when *SAMBA_INTERNAL* is selected                                                                                             |
    |                        |                |                                                                                                                                                                                           |
    +------------------------+----------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-   | Domain Forest Level    | drop-down menu | choices are *2000*,                                                                                                                                                                       |
-   |                        |                | *2003*,                                                                                                                                                                                   |
-   |                        |                | *2008*, or                                                                                                                                                                                |
-   |                        |                | *2008_R2*; refer to                                                                                                                                                                       |
+   | Domain Forest Level    | drop-down menu | choices are *2000*, *2003*, *2008*, *2008_R2*, *2012*, or *2012_R2*; refer to                                                                                                             |
    |                        |                | `Understanding Active Directory Domain Services (AD DS) Functional Levels <https://technet.microsoft.com/en-us/library/understanding-active-directory-functional-levels(WS.10).aspx>`_    |
```

Comment:
Use double underlines after an external link to prevent Sphinx from making an internal label.